frontend/src/pages/issue/IssueMemoSampleCsvPage.tsx

```diff
@@ -0,0 +1,107 @@
+import { ReactElement } from 'react'
+import { Link, Navigate, useNavigate, useParams } from 'react-router-dom'
+import {
+  Box,
+  Button,
+  Center,
+  Heading,
+  Icon,
+  Spinner,
+  Text,
+  useToast,
+  VStack,
+} from '@chakra-ui/react'
+import { Download } from '@styled-icons/boxicons-regular'
+import { Close } from '@styled-icons/material'
+import download from 'downloadjs'
+
+import { textStyles } from '~theme/textStyles'
+import { DASHBOARD_ROUTE } from '~constants/routes'
+import { useTemplate } from '~hooks/useTemplate'
+
+import { UIN_TOKEN, UIN_TYPE_TOKEN } from './constants'
+
+export const IssueMemoSampleCsvPage = (): ReactElement => {
+  const navigate = useNavigate()
+  const { templateId } = useParams()
+  const toast = useToast()
+
+  // Verify that the template exists
+  const { isLoading, isError, data: template } = useTemplate(Number(templateId))
+
+  const handleDownloadClick = async () => {
+    try {
+      let headers = template?.paramsRequired || []
+      // Params from backend should not have UIN_TOKEN and UIN_TYPE_TOKEN
+      headers = [UIN_TOKEN, UIN_TYPE_TOKEN, ...headers]
+      // Escape " and /
+      const content = headers
+        .map((h) => `"${h.replace(/\\"/g, '""')}"`)
+        .join(',')
```

Comment:
I might be wrong, but seems like the comment and regex don't match up 🤔 What is the purpose of replacing `\"` with `""`?

Reply:
Added a comment to clarify the escaping, hope it clarifies.

Reply:
Makes sense, lgtm!